app/modules/entities/components/deduplicate_authors_names.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import preq from 'lib/preq'
+  import Spinner from 'modules/general/components/spinner.svelte'
+  import { createEventDispatcher } from 'svelte'
+
+  const waitingForNames = fetchNames()
+  let names
+
+  async function fetchNames () {
+    const res = await preq.get(app.API.entities.duplicates)
+    names = res.names
```

Comment:
is `fetch` the new `grab` ? shall we stick to this verb for anything related to a API query ?